backend/app/expenses/service.py

```
@@ -378,10 +410,9 @@ async def update_expense(
                         await self._create_settlements_for_expense(
                             updated_expense, user_id
                         )
-                except Exception as e:
+                except Exception:
                     logger.error(
-                        f"Warning: Failed to recalculate settlements: {e}",
-                        exc_info=True,
+                        f"Warning: Failed to recalculate settlements", exc_info=True
```

Ruff (0.12.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)